README.md

```diff
@@ -63,7 +64,13 @@ kongAPI.createConfigurations()
         // Failure
     });
 
+
 ```
+
+## Note
+per each function (createApis,getAPIs,removeAPIs,removePlugins,createPlugins,getPluginsOfExistApi) headers object can be send, that will **overirde** the constructor headers.
```

Comment:
fix grammer:
`headers object can be sent, which will **override** the headers provided in the constructor (if any).`

Reply:
also - allow to set "headersToMask" list - and mask the values of these headers in logs